DataMigration/DataLayer/SQLConnect.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using DataMigration.PostgresDB;
+using System.IO;
+using System.Xml.Linq;
+using System.Xml.Serialization;
+using DataMigration.Logger;
+
+namespace DataMigration.DataLayer
+{
+    public class SqlConnect
+    {
+        private Logger.Logger _log=new Logger.Logger();
+
+        public void StartLogConsole(Logger.Logger logger)
+        {
+            _log = new Logger.Logger(logger);
+        }
+        public List<VanguardDoc> GetVanguardDocuments(string connectString,List<string> docPaths)
+        {
+            var vanguardDocs = new List<VanguardDoc>();
+            if (docPaths.Count == 0)
+            {
+                _log.WriteLog(LogLevel.Info, "No documents to read \n");
+                return vanguardDocs;
+            }
+            try
+            {
+                _log.WriteLog(LogLevel.Info, "Write documents in xml to make temporaly table in order to simplify detting data from DM_CONTENT table \n");
+                string resultStr;
+                using (var memoryStream = new MemoryStream())
+                {
+                    using (TextWriter streamWriter = new StreamWriter(memoryStream))
+                    {
+                        var xmlSerializer = new XmlSerializer(typeof(List<string>));
+                        xmlSerializer.Serialize(streamWriter, docPaths);
+                        resultStr = XElement.Parse(Encoding.ASCII.GetString(memoryStream.ToArray())).ToString(SaveOptions.OmitDuplicateNamespaces);
+                    }
+                }
+                using (var connection = new SqlConnection(connectString))
+                {
+                    connection.Open();
+                    try
+                    {
+                        using (var command = new SqlCommand
+                        {
+                            CommandType = CommandType.Text,
+                            Connection = connection,
+                            CommandTimeout = 0,
+                            CommandText = (@"
+                        declare @doc table (DocPath varchar(max)) 
+                        insert into @doc select tbl.col.value('.[1]', 'varchar(max)') from @XML.nodes('ArrayOfString/string') tbl(col) 
+                        select dmc.DM_ID, dmc.DMC_ID, dmc.DMC_PATH, dm.DEPT_ID
+                        from VG48215.DOC_MASTER dm
+                        join VG48215.DM_CONTENT dmc on dmc.DM_ID = dm.DM_ID
+                        join VG48215.DM_OCR_PROCESS ocr on ocr.DMC_ID = dmc.DMC_ID
```

Comment:
better to extract tenant code ("48215") into configuration and use appropriate value in all quieries